ForwardingDecision.cs

```diff
@@ -0,0 +1,43 @@
+/*
+Pax : tool support for prototyping packet processors
+Nik Sultana, Cambridge University Computer Lab, June 2016
+
+Use of this source code is governed by the Apache 2.0 license; see LICENSE.
+*/
+
+namespace Pax {
+
+  // A packet processor makes some sort of forwarding decision, in addition
+  // to possibly analysing/modifying the packet (and generating new packets).
+  // The forwarding decision makes clear whether the processor recommends to
+  // * drop the packet (i.e., nothing more is to be done with it)
+  // * forward to a single port
+  // * forward to multiple ports
+  //
+  // NOTE this idiom simulates using algebraic types in C#. It's values are
+  //      analysed using the Visitor pattern.
+  public abstract class ForwardingDecision {
```

Comment:
Ack, done at https://github.com/niksu/pax/pull/7/commits/7b706d68b2ef8845cb14fa588736a10a9e0f0bda and extended in https://github.com/niksu/pax/pull/7/commits/05f5f96f3e82d9f34a3f22bcaaafbed73003c831